services/web/project/templates/base.html

```diff
@@ -44,10 +44,14 @@
     </div>
 
 
+    <!-- JQuery 3.6.0 minified link -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
     <!--Bootstrap javascript link-->
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
         crossorigin="anonymous"></script>
+    <!-- question.html javascript link -->
+    <script type="text/javascript" src="{{ url_for('views.staticfile', folder='js', filename='question.js') }}"></script>
```

Comment:
It looks more like `question.js` is already our main.js since you include it into base template. If so just rename it :)

Reply:
question.js has code for question.html only. To keep things modular I extracted that code into a separate file. main.js should ideally contain code that touches on all views in the application (like the navbar perhaps for example). I think we should keep this modular approach and include the different JS files either here, or by specifying a Jinja2 module in the HTML file that it relates too like @VimFreeman siggested earliar today during our SCRUM meeting.

Reply:
> question.js has code for question.html only.

You include `question.js` on `base.html`. I would agree if you would include that into correct html :)

Reply:
Ah, well yes when you put it like that it makes sense. I'll move the script block to question.html